web/js/map-OpenLayers.js

```diff
@@ -115,6 +115,67 @@ var fixmystreet = fixmystreet || {};
         fixmystreet.markers.redraw();
       },
 
+      shortlist_multiple: function(ids, token) {
+        $.post("/my/planned/change_multiple", { ids: ids, token: token })
+        .done(function() {
+          var $itemList = $('.item-list'),
+              items = [];
+          $('.shortlisted-status').remove();
+
+          for (var i = 0; i < ids.length; i++) {
+            var problemId = ids[i],
+                $item = $itemList.find('#report-'+ problemId)
+                $form = $item.find('form'),
+                $submit = $form.find("input[type='submit']" );
+
+            $item.attr('data-lastupdate', new Date().toISOString());
```

Comment:
I'm not clear on what this is doing. `lastupdate` isn't usually updated on an add/remove shortlist, because the report hasn't changed. The lastupdate functionality should still operate okay without this, if the report is newly added, I think?